package/system/gpio-cdev/nu801/Makefile

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright (C) 2021 OpenWrt.org
```

Comment:
Unless this is put there by Christian, note that you are not OpenWrt, so you may not put this here.